src/ExtCmd.actor.cpp

```diff
@@ -406,6 +406,80 @@ struct DropCollectionCmd {
 };
 REGISTER_CMD(DropCollectionCmd, "drop");
 
+ACTOR static Future<int> internal_doRenameCollectionIndexesActor(Reference<DocTransaction> tr,
```

Comment:
This could return `Future<Void>` instead. So return statement would become `return Void();`